catalog/spatial/csw/spatial-csw-source-common/src/main/java/org/codice/ddf/spatial/ogc/csw/catalog/common/source/reader/GetRecordsMessageBodyReader.java

```diff
@@ -63,11 +63,10 @@
 
     private DataHolder argumentHolder;
 
-    public GetRecordsMessageBodyReader(Converter provider, CswSourceConfiguration configuration) {
+    public GetRecordsMessageBodyReader(Converter converter, CswSourceConfiguration configuration) {
         xstream = new XStream(new XppDriver());
         xstream.setClassLoader(this.getClass()
                 .getClassLoader());
-        GetRecordsResponseConverter converter = new GetRecordsResponseConverter(provider);
```

Comment:
We were passing in a converter for the sole purpose of creating yet another converter. This created unwanted dependency on spatial-csw-transformer (a bundle!) so I just changed what the argument being passed in represented. Now I create a GetRecordsResponsConverter in the blueprint and pass that in as the converter instead.